src/block.ts

```diff
@@ -4,44 +4,38 @@ export const geoBlockedCountryCodes = new Set(blocked_codes)
 
 export const isGeoBlocked = async () => {
   try {
-      const res = await fetch('https://geoip.gnosis.pm/json/')
+    const res = await fetch('https://geoip.gnosis.pm/json/')
 
       // this DOES NOT block even if the URL above starts returning 404
-      if (!res.ok) return false
+    if (!res.ok) return false
 
-      const { country_code } = await res.json()
+    const { country_code } = await res.json()
 
-      return geoBlockedCountryCodes.has(country_code)
-    } catch (error) {
-      console.error('Geo Blocking check is unavailable:', error.message || error, 'This is most likely a client side network connectivity issue - please retry connecting to the internet and refreshing the page.')
+    return geoBlockedCountryCodes.has(country_code)
+  } catch (error) {
+    console.error('Geo Blocking check is unavailable:', error.message || error, 'This is most likely a client side network connectivity issue - please retry connecting to the internet and refreshing the page.')
 
       // this does NOT block if there is a network error, e.g. URL is blocked
-      return false
-    }
+    return false
+  }
 }
 
-export const isNetBlocked = async () => {
-  if (typeof window === 'undefined') return false
-  const { hostname } = window.location
-    // allow anything when run locally
-  if (hostname === 'localhost' || hostname === '0.0.0.0') return false
-
-    // no walletextension detected, different error - download wallet error
-  if (!window.web3) return false
+export const isNetBlocked = async (idToAllow: string | number) => {
```

Comment:
You sure we ever allow only one network? I'm thinking maybe use a networkId array just in case